### ISSUE TITLE
Kinesis Event in Lambda contains only "kinesis" object data

### ISSUE THREAD

**Issue by raissaa** [localstack/localstack#2938]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

I'm currently using Localstack for integration tests and I'm trying to trigger a lambda (java 11) by putting events in a Kinesis stream. The lambda is triggered fine, but the event I get doesn't contain any fields other than some of the fields in "kinesis" object (ref: https://docs.aws.amazon.com/lambda/latest/dg/with-kinesis.html).
I also tried invoking the lambda with a json with the proper data and the same issue happens.
The generated event json seems fine.
May be related: https://github.com/localstack/localstack-java-utils/blob/master/src/main/java/cloud/localstack/LambdaExecutor.java#L71

## Expected behavior

The Kinesis event received in the lambda contains all fields related to the event as well.

```
{
    "Records": [
        {
            "kinesis": {
                "kinesisSchemaVersion": "1.0",
                "partitionKey": "1",
                "sequenceNumber": "49590338271490256608559692538361571095921575989136588898",
                "data": "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0Lg==",
                "approximateArrivalTimestamp": 1545084650.987
            },
            "eventSource": "aws:kinesis",
            "eventVersion": "1.0",
            "eventID": "shardId-000000000006:49590338271490256608559692538361571095921575989136588898",
            "eventName": "aws:kinesis:record",
            "invokeIdentityArn": "arn:aws:iam::123456789012:role/lambda-role",
            "awsRegion": "us-east-2",
            "eventSourceARN": "arn:aws:kinesis:us-east-2:123456789012:stream/lambda-stream"
        }
    ]
}
```

## Actual behavior

The Kinesis event only has the "kinesis" object filled with the *some* of the correct data.

```
{
    "Records": [
        {
            "kinesis": {
                "partitionKey": "1",
                "data": "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0Lg==",
                "approximateArrivalTimestamp": 1545084650.987
            },
        }
    ]
}
```

# Steps to reproduce

## Command used to start LocalStack

Docker compose using the cloudformation template

**Comment by raissaa** [comment]
@usmangani1 I think the issue is on the executor side: https://github.com/localstack/localstack-java-utils/blob/master/src/main/java/cloud/localstack/LambdaExecutor.java#L71

**Comment by whummer** [comment]
Thanks for reporting @raissaa . This should be fixed in #2946 - can you please do a `docker pull localstack/localstack` and give it another try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by raissaa** [comment]
@whummer thanks, but it's still happening, as I said in the previous comment I think it's the executor that's filtering out the data
